css/content.css

```diff
@@ -639,6 +639,20 @@
     white-space: normal;
 }
 
+#app-content .body table {
+  border-collapse: collapse;
+}
+
+#app-content .body table th {
+  font-weight: bold;
+}
+
+#app-content .body table th,
+#app-content .body table td {
+  border: 1px solid #888;
```

Comment:
Could this use one of the variables like `var(--color-border-lighter);` to fit in the themes?

Reply:
Sure, I'll look at the variables and see what fits. I also noticed the indentation is wrong so I'll fix that.

Reply:
Neither `--color-border` nor `--color-border-dark` are easily visible. The closest I can find is `--color-text-lighter`, is that ok?

Reply:
Yeah, that's fine to me

Reply:
Screenshots with `color-text-lighter`:

![obrazek](https://user-images.githubusercontent.com/3685160/168024444-c15a1dd7-1e31-4e8a-a2e0-1fd896db566f.png)

![obrazek](https://user-images.githubusercontent.com/3685160/168024341-156e3487-c896-43ef-af1d-7f95ab100b29.png)